app/app/urls.py

```diff
@@ -443,6 +443,7 @@
     ),
 
     # View Bounty
+    # TODO geri: should we drop the 2 URL patterns below???
```

Comment:
Is this something which needs to be addressed now ? 
would it make more sense to track this in a ticket as opposed to the code ?

Reply:
I was reluctant to remove this to avoid breaking the links that users have ....
I will write a proper comment here.